server/src/dtos/user.dto.ts

```diff
@@ -62,7 +62,6 @@ export class UserAdminCreateDto {
   @Transform(toEmail)
   email!: string;
 
-  @IsNotEmpty()
```

Comment:
I think this just allows empty strings, but it should probably have Optional({ nullable: true }) and still keep the is not empty. We should also test that you cannot login with an empty string of password login is enabled still

Reply:
We have set password to be string in the user entity. So would it be okay to insert null values?
https://github.com/immich-app/immich/blob/3f663106e8093824617128d7558423284c4741cd/server/src/entities/user.entity.ts#L33

And during login we have added check for not allowing empty password
https://github.com/immich-app/immich/blob/3f663106e8093824617128d7558423284c4741cd/server/src/services/auth.service.ts#L329

Reply:
Oh, I see that we don't allow null in the database. We can change that later I think. We'll allow empty now, and move to `null` in the future.